### PR TITLE
Weblate: Rename Win Installer files to fit standard

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**

Weblate currently doesn't detect the correct language of the Windows installer files. This renames the problematic files to fit the .ts language files.

CHANGELOG: SKIP
**Context: Fixes an issue?**

Related to: https://github.com/jamulussoftware/jamulus/issues/2758#issuecomment-1236373032
**Does this change need documentation? What needs to be documented and how?**

No.

**Status of this Pull Request**

Waiting on second review.

**What is missing until this pull request can be merged?**

Since this can't be tested unless it's merged, I assume only 2 reviews.

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [x] I've filled all the content above

### CODE REVIEW

File: src/translation/wininstaller/installerlng.nsi
Comment (pljones) [inline src/translation/wininstaller/installerlng.nsi:33]: pt_PT as we have a separate pt_BR, then?

Comment (ann0see) [reply src/translation/wininstaller/installerlng.nsi]: Done

**Comment by hoffie** [comment]
Maybe I have missed the relevant context, but what's the guideline when we use aa.nsi and when aa_BB.nsi? Wouldn't it be more consistent to use aa_BB.nsi everywhere, similar to .ts files?
Otherwise it would be hard to decide how to name new files, if the only guideline is "will Weblate detect it"?

**Comment by ann0see** [comment]
We can of course rename all of them.

**Review by hoffie — APPROVED** [review]
Thanks. This looks more consistent to me now. :)

**Comment by ann0see** [comment]
> Not a big fan of how it looks in Weblate, but at least it is available for translation and should be easy to understand as well.

No. But since it's not a standard, I doubt anything better works.

**Comment by ann0see** [comment]
Feel free to merge this after a second approval.

**Comment by pljones** [comment]
Is there really no documentation explaining how the winstaller translation work?  Oh well.  We should have some...  Separate issue.

**Review by pljones — APPROVED** [review]

**Pull request merged**
